apps/src/webapp/webapp.js

```diff
@@ -505,6 +506,14 @@ Webapp.init = function(config) {
     }
   }
 
+  var viewDataButton = document.getElementById('viewDataButton');
+  dom.addClickTouchEvent(viewDataButton, Webapp.onViewData);
+
+  if (!StudioApp.hideSource) {
```

Comment:
If you only added this button when the debugButtons flag was set (per my earlier comment), you could rewrite this a little.